modules/api/pkg/ee/clusterbackup/storage-location/provider.go

```diff
@@ -110,57 +141,78 @@ func (p *BackupStorageProvider) CreateUnsecured(ctx context.Context, cbslName, p
 		},
 		Key: "cloud-credentials",
```

Comment:
I would recommend to move this into a global const, since we are using this in multiple places in this provider